builtin/game/chatcommands.lua

```diff
@@ -763,14 +763,20 @@ core.register_chatcommand("days", {
 
 core.register_chatcommand("shutdown", {
 	description = "Shutdown server",
-	params = "[reconnect] [message]",
+	params = "[delay_in_seconds(0..inf)] [reconnect] [message]",
```

Comment:
I think delay=-1 cancels the shutdown; this should say something like:

[delay_in_seconds(0..inf) or -1 for cancel] [reconnect] [message]